### PR TITLE
Make Prefill Donation Inputs Distinguishable In Dark Mode

### PR DESCRIPTION
**Pull request by sampoder**
Before:

Screenshot 2023-08-23 at 11 46 10 AM

After:

Screenshot 2023-08-23 at 11 23 09 AM

I thought this would help people work out what's an input!

### CODE REVIEW

**Review by cjdenio — DISMISSED** [review]
For consistency with other modals in the interface, you just need to add the `modal--scroll bg-snow` classes to the modal element.
https://github.com/hackclub/bank/blob/c3e30d30e7d8bae5ff00926cec48e7e8531b4115/app/views/events/donation_overview.html.erb#L53

**Comment by sampoder** [comment]
Oh that's perfect!!

File: app/assets/stylesheets/_dark.scss [hunk 1]
```diff
@@ -28,6 +28,12 @@ html[data-dark='true'] {
       border-color: map-get($palette, info);
     }
   }
+  
+  .bg-darker {
+    background: $dark!important;
+    color: map-get($palette, white);
+    border-color: $dark!important;
+  }
```
Comment (sampoder) [inline app/assets/stylesheets/_dark.scss]:
```suggestion
```

File: app/views/events/donation_overview.html.erb [hunk 2]
```diff
@@ -27,7 +27,7 @@
             Embed code
           <% end %>
           <%= link_to "#",
-            data: { behavior: "modal_trigger", modal: "prefilled_link" } do %>
+            data: { behavior: "_trigger", : "prefilled_link" } do %>
```
Comment (sampoder) [inline app/views/events/donation_overview.html.erb]:
```suggestion
            data: { behavior: "modal_trigger", modal: "prefilled_link" } do %>
```

I think I accidentally changed this

**Comment by sampoder** [comment]
awesome! @cjdenio got rid of that extra class

**Review by YodaLightsabr — APPROVED** [review]
LGTM!

**Pull request merged**